search/filters/FulltextFilter.php

```diff
@@ -73,5 +73,18 @@ public function getDbName() {
 
 		return parent::getDbName();
 	}
+	
+	private function prepareColumns($columns) {
+		$class = '';
+		$parts = explode(',', $columns);
+		array_walk($parts, function(&$col, $key) use (&$class) {
+			$name = trim($col, " \t\n\r\0\x0B\"");
+			if (empty($class)) {
```

Comment:
Also check out DBSchemaManager::explodeColumnString for an existing implementation of this index spec parsing.

I wish I made that public... oh well. :)